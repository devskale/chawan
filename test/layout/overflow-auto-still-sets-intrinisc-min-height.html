<!DOCTYPE html>
<div style="height: 2em; display: flex; flex-direction: column">
<div>
<div style="overflow: auto; height: 2ch">
<pre>
test
test
test
test
test
test
</pre>
</div>
</div>
<div>

footer footer footer footer
