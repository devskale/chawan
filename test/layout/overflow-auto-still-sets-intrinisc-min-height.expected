
test
test
test
test
test
test

footer footer footer footer
